go/cmd/coordinator/cmd.go

```diff
@@ -54,7 +54,7 @@ func init() {
 	// Query service memberlist
 	Cmd.Flags().StringVar(&conf.QueryServiceMemberlistName, "query-memberlist-name", "query-service-memberlist", "Query service memberlist name")
 	Cmd.Flags().StringVar(&conf.QueryServicePodLabel, "query-pod-label", "query-service", "Query pod label")
-	Cmd.Flags().DurationVar(&conf.WatchInterval, "watch-interval", 60*time.Second, "Watch interval")
+	Cmd.Flags().DurationVar(&conf.WatchInterval, "watch-interval", 5*time.Second, "Watch interval")
```

Comment:
cc @Ishiihara - i think we can override this in deployment, but for local its nice to have it low. wdyt?